automation/src/test/scala/org/broadinstitute/dsde/workbench/leonardo/LeonardoSuite.scala

```diff
@@ -61,34 +80,54 @@ trait GPAllocUtils extends BillingFixtures with LeonardoTestUtils {
                                                       BillingProject.BillingProjectRole.User)(hermioneAuthToken)
       )
       _ <- loggerIO.info(s"Billing project claimed: ${claimedBillingProject.projectName}")
-    } yield GoogleProject(claimedBillingProject.projectName)
+      workspaceName = UUID.randomUUID().toString
+      _ <- IO(
+        Orchestration.workspaces.create(claimedBillingProject.projectName, workspaceName)(ronAuthToken)
+      )
+      // Testing with: sbt "testOnly *RuntimeAutopauseSpec"
+      workspaceDetails <- IO(
+        Rawls.workspaces.getWorkspaceDetails(claimedBillingProject.projectName, workspaceName)(ronAuthToken)
+      )
+      googleProjectId <- IO(
+        workspaceDetails.parseJson.asJsObject
+          .getFields("workspace")
+          .flatMap(workspace => workspace.asJsObject.getFields("googleProjectId"))
```

Comment:
we're reverting `googleProjectId` back to `googleProject` so that it is backwards compatible